cli_tools/gce_windows_upgrade/upgrader/upgrader.go

```diff
@@ -92,6 +95,11 @@ type upgrader struct {
 	*InputParams
 	*derivedVars
 
+	// Logger across all workflows
+	Logger      daisy.Logger
+	cloudLogger *logging.Logger
+
```

Comment:
I'd recommend refactoring this so that your business logic (this class) doesn't know or care about two different loggers. Instead, create *one* logger that you pass to each daisy workflow, and the backend of that logger knows how to write to the correct places.

This is my plan for image import.

Reply:
removed all log changes